components/brave_rewards/core/endpoint/promotion/post_suggestions_claim/post_suggestions_claim.cc

```diff
@@ -85,16 +84,19 @@ void PostSuggestionsClaim::Request(const credential::CredentialsRedeem& redeem,
       base::BindOnce(&PostSuggestionsClaim::OnRequest, base::Unretained(this),
                      std::move(callback));
 
-  auto headers =
-      util::BuildSignHeaders("post /v2/suggestions/claim", payload,
-                             wallet->payment_id, wallet->recovery_seed);
-
   auto request = mojom::UrlRequest::New();
   request->url = GetUrl();
   request->content = payload;
-  request->headers = headers;
   request->content_type = "application/json; charset=utf-8";
```

Comment:
Do we no longer need to sign the headers here?

Reply:
From what I'm understanding here, this previously signed all the headers before and now we're just signing the digest of the content body. This seems like we're effectively signing only part of the message now with this change rather than signing all of the headers. Am I misunderstanding how the code works here?

Reply:
I just realized my mistake when looking at this again. You're correct this still matches.